packages/web-react/tsconfig.json

```diff
@@ -24,7 +24,7 @@
     "noImplicitAny": true,
     "esModuleInterop": true,
     "typeRoots": ["./node_modules/@types"],
-    "lib": ["es2015", "dom", "dom.iterable"]
+    "lib": ["es2015", "es2021.string", "dom", "dom.iterable"]
```

Comment:
Please, don't. See https://github.com/lmc-eu/spirit-design-system/pull/538/files#r1015170339 and remove this.